docs/pages/eas/metadata/config.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Configuring EAS Metadata
+sidebar_title: store.config.json
+---
+
+import { Callout } from '~/ui/components/Callout';
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+EAS Metadata is configured by a **store.config.json** file in the _root of your project_.
+
+You can configure the path or name of the store config file with the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.
+Besides the default JSON format, EAS Metadata also supports more dynamic config using JavaScript files.
+
+## Static store config
+
+The default store config type for EAS Metadata is a simple JSON file.
+The example below shows an example store config with basic App Store information written in English (U.S.).
+
+You can find all configuration options in the [store config schema](./schema.md).
+
+> If you have the [VS Code Expo plugin](https://github.com/expo/vscode-expo#readme) installed, you get auto-complete, suggestions, and warnings for **store.config.json** files.
+
+<CodeBlocksTable tabs={['store.config.json']}>
+
+```json
+{
+  "configVersion": 0,
+  "apple": {
+    "info": {
+      "en-US": {
+        "title": "Awesome App",
+        "subtitle": "Your self-made awesome app",
+        "description": "The most awesome app you've ever seen",
+        "keywords": ["awesome", "app"],
+        "marketingUrl": "https://example.com/en/promo",
+        "supportUrl": "https://example.com/en/support",
+        "privacyPolicyUrl": "https://example.com/en/privacy"
+      }
+    }
+  }
+}
+```
+
+</CodeBlocksTable>
+
+
+## Dynamic store config
+
+Metadata like the **copyright notice** should contain the current year, with EAS Metadata this can be automated.
```

Comment:
```suggestion
At times, Metadata fields can benefit from dynamic values. For example, the Metadata **copyright notice** should contain the current year. This can be automated with EAS Metadata.
```

Reply:
```suggestion
At times, Metadata properties can benefit from dynamic values. For example, the Metadata **copyright notice** should contain the current year. This can be automated with EAS Metadata.
```

I think I referred to these as "metadata properties", as in object properties. Will add your suggestion, but with `fields -> properties`, if that's ok with you 😄